### DISCUSSION TITLE
#Blog 13.10.2024

### DISCUSSION THREAD

**Discussion by Ferreidi** [Anastylos/Jump-and-Run#5] [Blog]
Blog 13.10.24

1.  Unser Team besteht aus Michael (zuständig für Tech Lead und Playerdesign), Lukas (zuständig für Gegnerdesgin), Don (zuständig für Leveldesign), Christoph (der Organisator) und Dinis (zuständig für das Progress-design). 

2.  Gemeinsam möchten wir das Projekt verwirklichen ein Jump and Run Spiel zu entwickeln. Dafür benutzen wir Godot. Im Spiel soll man verschiedene Gegner bekämpfen können und Belohnungen erhalten, wenn man diese besiegt. Außerdem soll es Errungenschaften geben die als Anreiz dienen bestimmte Items im Spiel freizuschalten. 

3.  Wir verwenden Godot für die Programmierung des Projektes. Ebenfalls nutzen wir diverse Features von GitHub wie z.B. GitHub-projects. 

4.  Wir treffen uns wöchentlich am Mittwoch um die aktuellen Stände der einzelnen Mitglieder der Gruppe auszutauschen. Ebenfalls findet das Daily Montags, Mittwochs und Freitags statt. Für den Sprint wird fürs erste 1 Woche Zeit entschieden und wir nutzen als Scrum-Setup ausschließlich GitHub.

5.  In unserem Sprint haben wir das Ziel gesetzt, schon ein paar Schritte zur Vorbereitung des Projektes durchzuführen. Wie unten im Screenshot sehen wir schon die Punkte die im Progress sich befinden. Unter anderem hat Michi sich das Tutorial zu Godot angeguckt und ist dabei das Main-Build für das Spiel zu erstellen. Dabei hat Dinis die Aufgabe den ersten Blog zu schreiben, damit dieser bis Dienstag fertig ist. Christoph ist dabei momentan die ganzen GitHub Projects zu erstellen, damit wir ungestört GitHub nutzen können ohne jegliche Schwierigkeiten oder Probleme. Lukas ist momentan dabei das Tutorial zur Godot-Engine anzugucken, während Don die Aufgabe hat ein Gamedesign-Dokument zu erstellen, wo unter anderem es um die Level, Schwierigkeiten und Gegner des Spiels geht.

6.  
      ![Kanban-Board_13-10-2024] (https://github.com/user-attachments/assets/16fc6034-4a07-4507-868c-527d8ddd7311)
      ![Übersicht einer Aufgabe im Kanban-Board] (https://github.com/user-attachments/assets/7ae6cdbc-c987-4a87-baf0-827da5f27e32)

**Comment by MB-stack06** [comment]
Euer Projekt klingt unglaublich spannend! Die Idee, ein Jump-and-Run-Spiel zu entwickeln, in dem man Gegner bekämpfen und Belohnungen sowie Errungenschaften sammeln kann, ist super motivierend. Besonders toll finden wir, wie ihr klare Ziele setzt und bereits mit der Vorbereitung voll durchstartet. Es klingt so, als ob euer Spiel eine Menge Potenzial hat, die Spieler mit abwechslungsreichen Herausforderungen zu fesseln. Wir bin schon sehr gespannt auf die nächsten Updates und freuen uns darauf zu sehen, wie sich das Spiel weiterentwickelt. 

Viel Erfolg bei der Umsetzung
Euer GymTrackerX-Team

**Comment by ldcdorn** [comment]
Super, dass ihr die Rollen so konkret aufgeteilt habt! Würde mich interessieren, warum ihr euch für Godot als Engine entschieden habt. Finde es auch gut, dass ihr schon an ein Gamedesign Dokument denkt; freu mich schon, da mal reinzuschauen.
Ich persönlich würde euch den Vorschlag machen, den Product Backlog von euerem Sprint Backlog klar zu trennen. Genauere Informationen findet ihr im [Scrum Guide](https://www.scrum.org/learning-series/what-is-scrum/).

Unser Team freut sich schon extrem, das Spiel anzutesten :wink: ,  auf welchen Platformen wird es den erscheinen und was ist der voraussichtliche "Release Date"?

Viel Grüße
Project HÆLTH
